src/OrbitBase/NotFoundOrTest.cpp

```diff
@@ -40,6 +40,17 @@ TEST(NotFoundOr, GetNotFoundMessage) {
   EXPECT_EQ(GetNotFoundMessage(not_found_or_int), message);
 }
 
+TEST(NotFoundOr, GetFound) {
+  NotFoundOr<int> not_found_or_int = NotFound{"message"};
+  EXPECT_DEATH(std::ignore = GetFound(not_found_or_int), "Check failed");
+
+  not_found_or_int = 5;
+  EXPECT_EQ(GetFound(not_found_or_int), 5);
+
+  not_found_or_int = 6;
+  EXPECT_EQ(GetFound(not_found_or_int), 6);
+}
+
 TEST(NotFoundOr, MoveOnlyType) {
```

Comment:
I think `GetFound` should be tested in `MoveOnlyType` as well.

Reply:
Yeah, I "conveniently" did not do that, because it was not possible with the copy mechanism. I added a const ref version now.